bears/general/CPDBear.py

```diff
@@ -91,7 +89,8 @@ def run(self, language: str, minimum_tokens: int=20,
         files = ','.join(self.file_dict.keys())
         executable = which('pmd') or which('run.sh')
         arguments = ('bash', executable, 'cpd', '--skip-lexical-errors',
-                     '--minimum-tokens', str(minimum_tokens),
+                     '--minimum-tokens', str(
+                            Redundancy.Clone.min_clone_tokens.default),
```

Comment:
uhm coala doesn't populate those yet properly right so we can't use it that way yet